docs/quickstarts/javascript.mdx

```diff
@@ -60,13 +60,11 @@ To use the ClerkJS package, you'll need your **Publishable Key** and your **Fron
 
 ```sh filename=".env.local"
 REACT_APP_CLERK_PUBLISHABLE_KEY={{pub_key}}
-CLERK_FRONTEND_API=[your-domain].clerk.accounts.dev
+CLERK_PUBLISHABLE_KEY=[your-domain].clerk.accounts.dev
```

Comment:
That change is not correct, we should rename this to `CLERK_FRONTEND_API_URL` as that should be used in the script tag to load the package